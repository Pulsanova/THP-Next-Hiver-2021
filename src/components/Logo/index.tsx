import LogoImage from './logo.svg';

const Logo = () => (
    <img className="Logo" src={LogoImage} />
);

export default Logo;
