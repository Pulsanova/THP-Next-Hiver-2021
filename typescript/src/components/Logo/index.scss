.Logo {
    max-width: 300px;
}
