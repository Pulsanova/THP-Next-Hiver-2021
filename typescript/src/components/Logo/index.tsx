import './index.scss';
import LogoImage from './logo.svg';

const Logo = () => (
    <img className="Logo" src={LogoImage} alt="logo" />
);

export default Logo;
