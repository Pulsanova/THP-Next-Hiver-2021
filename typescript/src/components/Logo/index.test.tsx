import React from 'react';
import { render } from '@testing-library/react';
import Logo from './';

describe('<Logo /> component', () => {
    render(<Logo />);
    const img = document.querySelector('img');

    test('matches HTML snapshot correctly', () => {
        expect(img).toMatchSnapshot();
    });
});
