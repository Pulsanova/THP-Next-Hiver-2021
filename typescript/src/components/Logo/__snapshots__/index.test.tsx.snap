// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> component matches HTML snapshot correctly 1`] = `
<img
  alt="logo"
  class="Logox"
  src="logo.svg"
/>
`;
